unsloth/chat_templates.py

```diff
@@ -1190,7 +1193,21 @@
 DEFAULT_SYSTEM_MESSAGE["qwen3"] = None # No default system message for Qwen-3
 pass
 
-def _change_system_message(template: str, type_chat_template: str, system_message: str = None):
+def _change_system_message(template: str, type_chat_template: str, system_message: str = None) -> None:
```

Comment:
For underscore type functions which are not user facing, please reduce docs length